extensions/wikia/ContentReview/controllers/ContentReviewApiController.class.php

```diff
@@ -207,13 +208,55 @@ public function showTestModeNotification() {
 		$notification = wfMessage( 'content-review-test-mode-enabled' )->escaped();
 		$notification.= Xml::element(
 			'a',
-			[ 'class' => 'content-review-test-mode-disable', 'href' => '#' ],
-			wfMessage( 'content-review-test-mode-disable' )->plain()
+			[ 'class' => 'content-review-module-test-mode-disable', 'href' => '#' ],
+			wfMessage( 'content-review-module-test-mode-disable' )->plain()
 		);
 
 		$this->notification = $notification;
 	}
 
+	/**
+	 * Prepares parts for rendering status module
+	 * Returns data required by ContentReviewModule.mustache
+	 * @throws MWException
+	 */
+	public function renderStatusModal() {
+		global $wgCityId;
```

Comment:
You can use `$this->wg->` for retrieving global variables in a context of a controller.